dpctl-capi/tests/test_sycl_device_interface.cpp

```diff
@@ -275,6 +275,86 @@ TEST_P(TestDPCTLSyclDeviceInterface, Chk_GetPreferredVectorWidthHalf)
     }
 }
 
+TEST_P(TestDPCTLSyclDeviceInterface, Chk_GetImage2dMaxWidth)
+{
+    DPCTLSyclDeviceRef DRef = nullptr;
+    size_t image_2d_max_width = 0;
+    EXPECT_NO_FATAL_FAILURE(DRef = DPCTLDevice_CreateFromSelector(DSRef));
+    if (!DRef)
```

Comment:
I changed the test cases in master. I moved the `DPCTLDevice_CreateFromSelector` into the fixture constructor and the `GTEST_SKIP` to the `SetUp()`. Reduces the redundant code in each test case. Please update these test cases in the same way.